packages/sdk-snippets/README.md

```diff
@@ -0,0 +1,87 @@
+# @readme/metrics-sdk-snippets
+
+Generate code snippets for ReadMe Metrics SDKs in multiple languages.
+
+[![Build](https://img.shields.io/github/workflow/status/readmeio/metrics-sdks/snippets.svg)](https://github.com/readmeio/metricds-sdks) [![](https://img.shields.io/npm/v/@readme/metrics-sdk-snippets)](https://npm.im/@readme/metrics-sdk-snippets)
+
+[![](https://d3vv6lp55qjaqc.cloudfront.net/items/1M3C3j0I0s0j3T362344/Untitled-2.png)](https://readme.io)
+
+This library was built for [ReadMe's Metrics](https://readme.com/metrics) onboarding flow.
+
+## Installation
+
+```sh
+npm install --save @readme/metrics-sdk-snippets
+```
+
+## Usage
+
+```js
+import { MetricsSDKSnippet } from '@readme/metrics-sdk-snippets';
+
+const { convert } = new MetricsSDKSnippet([
+  {
+    name: 'petstore_auth',
+    default: 'default-key',
```

Comment:
Are we able to support `default === ''`?

Reply:
I'll add some tests.